landingzones/lz-platform-connectivity-hub-azfw/azfw-policy/azure-firewall-policy.bicep

```diff
@@ -760,10 +760,132 @@ resource policy 'Microsoft.Network/firewallPolicies@2021-02-01' = {
     }
   }
 
+  // AKS required FQDNs 
+  // https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic
+  resource AKSCollectionGroup 'ruleCollectionGroups@2021-02-01' = {
+    dependsOn: [
+      windowsCollectionGroup
+    ]
+    name: 'Aks'
+    properties: {
+      priority: 1200
+      ruleCollections: [
+        {
+          ruleCollectionType: 'FirewallPolicyFilterRuleCollection'
+          name: 'Ubuntu NTP'
+          action: {
+            type: 'Allow'
+          }
+          priority: 100
+          rules: [
+            {
+              ruleType: 'NetworkRule'
+              name: 'Ubuntu NTP - FQDN'
+              destinationAddresses: [
+                'ntp.ubuntu.com'
+              ]
+              destinationPorts: [
+                '123'
+              ]
+              ipProtocols: [
+                'UDP'
+              ]
+              sourceAddresses: [
+                '*'
+              ]
+            }
+          ]
+        }
+        {
+          ruleCollectionType: 'FirewallPolicyFilterRuleCollection'
+          name: 'AKS Azure Global required FQDNs'
+          action: {
+            type: 'Allow'
+          }
+          priority: 150
+          rules: [
+            {
+              ruleType: 'ApplicationRule'
+              name: 'AKS required FQDNs'
+              targetFqdns: [
+                '*.hcp.canadacentral.azmk8s.io'
+                'mcr.microsoft.com'
+                '*.data.mcr.microsoft.com'
+                'management.azure.com'
+                'login.microsoftonline.com'
+                'packages.microsoft.com'
+                'acs-mirror.azureedge.net'
+                'canadacenral.dp.kubernetesconfiguration.azure.com'
```

Comment:
Is this right?  `canadacenral.dp.kubernetesconfiguration.azure.com` where canadacentral is spelled wrong.